src/main/resources/templates/index.html

```diff
@@ -1,59 +1,65 @@
 <!DOCTYPE html>
 <html>
-<head>
-<meta charset="ISO-8859-1"/>
-<title>Feel Free Cincinnati</title>
-<link href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
-<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js"></script>
-<style>
-header {color: #00FFFF; background-color: #000000;}
-h1 {font-size: 50px}
-body {background-color: #BBBBBB}
-footer {text-align: center; font-size:75%; font-style: italic}
-</style>
-</head>
-<body>
-<header><div class="text-center"><h1>Feel Free Cincinnati</h1></div>
-<div class="text-center">
-<div class="custom-control custom-radio custom-control-inline">
-  <input type="radio" id="eventRdo" name="customRadioInline1" class="custom-control-input">
-  <label class="custom-control-label" for="eventRdo">Find an Event</label>
-</div>
-<div class="custom-control custom-radio custom-control-inline">
-  <input type="radio" id="adventureRdo" name="customRadioInline1" class="custom-control-input">
-  <label class="custom-control-label" for="adventureRdo">Go on an Adventure</label>
-</div></div>
+	<head>
+		<meta charset="ISO-8859-1"/>
+		
+		<title>Feel Free Cincinnati</title>
+		
+		<link href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+		<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js"></script>
+		
+		<style>
+		header {color: #00FFFF; background-color: #000000;}
+		h1 {font-size: 50px}
+		body {background-color: #BBBBBB}
+		footer {text-align: center; font-size:75%; font-style: italic,position: fixed; left: 0px; bottom: 0px; height: 30px; width: 100%; background: #999;}

```

Comment:
While refactoring, this is a good opportunity to move these css styles to an external stylesheet.